src/index.js

```diff
@@ -10,7 +10,7 @@ export default function fileLoader(content) {
 
   const query = loaderUtils.getOptions(this) || {};
   const configKey = query.config || 'fileLoader';
-  const options = this.options[configKey] || {};
+  const options = this.options && this.options[configKey] || {};
```

Comment:
@jquense i am use `file-loader` without `LoaderOptionsPlugin`, can your provide repo where it is not works?